### PR TITLE
Feature/sign up form

### PR DESCRIPTION
**Pull request by isivak-fevo**

### CODE REVIEW

**Comment by cyberduck1** [comment]
@sivaki 
Look good.
I would probably move the login form into a separate file as well?!?
Then the if else will toggle between 2 components?

File: client/src/components/login/signUp/signUp.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+          lastName: lastName,
+          email: email,
+          password: password
+        }
+      })
+    }}>
+        <Grid container spacing={2}>
+          <Grid item xs={12} sm={6}>
+            <TextField
+              autoComplete="fname"
```
Comment (Evkaaa) [inline client/src/components/login/signUp/signUp.tsx]: Do we want to use this kind of long formatting?

Comment (cyberduck1) [reply client/src/components/login/signUp/signUp.tsx]: It's just a rule we can change it to whatever is best.

Comment (isivak-fevo) [reply client/src/components/login/signUp/signUp.tsx]: I like this style more, to be honest, but I don't mind, I can adjust it as you (or as most of us) wish 😄

Comment (cyberduck1) [reply client/src/components/login/signUp/signUp.tsx]: Actually, if you will change the formating it might not pass lint tests.
But you can try :)

**Comment by isivak-fevo** [comment]
@cyberduck0 
I also wanted to put the login to the separated file, I'd also prefer to change class component to functional component. I can do it and continue in this branch if you want 🙂

File: client/src/components/login/signUp/signUp.tsx (hunk 1 above)
Comment (Evkaaa) [reply client/src/components/login/signUp/signUp.tsx]: To be honest, I don't like this long formatt. It's kinda unreadable for me. I agree we should agree on some formatt to have it at least the similar way ;)

Comment (isivak-fevo) [reply client/src/components/login/signUp/signUp.tsx]: Ok, I'll resolve this github conversation and we can discuss it in slack. 🙂

**Comment by cyberduck1** [comment]
I think, finish with it in this branch and we will merge after you're done

File: client/src/components/login/signUp/signUp.tsx (hunk 1 above)
Comment (Evkaaa) [reply client/src/components/login/signUp/signUp.tsx]: Format is so important for lint? I have never used it, so I have no idea.

Comment (cyberduck1) [reply client/src/components/login/signUp/signUp.tsx]: Yeah lint is like standardized formatting + TS checks
But it's built on top of rules which we can change.
We just need to agree on some standards.

**Comment by cyberduck1** [comment]
CI Test

**Pull request merged**